barretenberg/cpp/src/barretenberg/dsl/acir_proofs/acir_composer.cpp

```diff
@@ -152,9 +213,12 @@ std::string AcirComposer::get_solidity_verifier()
  * @param proof
  * @param num_inner_public_inputs - number of public inputs on the proof being serialized
  */
-std::vector<barretenberg::fr> AcirComposer::serialize_proof_into_fields(std::vector<uint8_t> const& proof,
-                                                                        size_t num_inner_public_inputs)
+std::vector<barretenberg::fr> AcirComposer::serialize_proof_into_fields(
```

Comment:
Maybe separate discussion but, can we just standardise this format? Is the recursion format significantly less efficient or something?

Reply:
Ideally I would like to use the same format for non-recursion and for recursion (bytes), then we could get rid of this method, I'm not sure what the efficiency loss is here though